lua/tsc/init.lua

```diff
@@ -92,16 +99,14 @@ M.run = function()
       return
     end
 
-    local notify_opts = { title = "TSC" }
-
     if notify_record ~= nil then
-      notify_opts = vim.tbl_extend("force", { replace = notify_record.id }, notify_opts)
+      options.notify.options = vim.tbl_extend("force", options.notify.options, { replace = notify_record.id })
     end
```

Comment:
Lets not reassign/mutate the `config` object. Instead lets add back `local notify_opts = { title = "TSC" }` above this and then for the `if` statement lets do:

```lua
if notify_record ~= nil then
  notify_opts = vim.tbl_deep_extend("force", notify_opts, { replace = notify_record.id }, config.notify.options)
end
```